nmt_chainer/beam_search.py

```diff
@@ -82,8 +82,8 @@ def update_next_lists(num_case, idx_in_case, new_cost, eos_idx, new_state_ensemb
             attn_summed /= len(attn_ensemble)
             next_attentions_list.append(current_attentions[num_case] + [attn_summed])
             
-def compute_next_lists(new_state_ensemble, new_scores, beam_width, eos_idx, current_translations, finished_translations, 
-      current_attentions, attn_ensemble, force_finish = False, need_attention = False):
+def compute_next_lists(new_state_ensemble, new_scores, beam_width, beam_size, eos_idx, current_translations, finished_translations, 
+      current_attentions, attn_ensemble, force_finish = False, need_attention = False, consider_beam_size_when_pruning = False):
 
```

Comment:
Also, I am actually finding the term "beam_size" a bit ambiguous and easy to confuse with the concept of "beam_width" already used in the code. Even if it was the term in the paper, I would prefer something else. Maybe "beam_pruning_margin".